source/MRViewer/ImGuiMenu.h

```diff
@@ -291,6 +291,8 @@ class MRVIEWER_CLASS ImGuiMenu : public MR::ViewerPlugin,
 
     void draw_open_recent_button_();
 
+    // A virtual function for drawing of the dialog with shortcuts. It can be overriden in the inherited classes
+    MRVIEWER_API virtual void drawShortcutsWindow_();
```

Comment:
mb better add non-blocking state plugin that shows shortcuts, and setup it in setupShortcuts_ itself